app/src/views/features/sessions/SessionViewer/DraftSessionViewer.tsx

```diff
@@ -185,6 +226,20 @@ const DraftSessionViewer: React.FC = () => {
     });
   };
 
+  const onCheckboxChange = (checkboxValues: CheckboxValueType[]) => {
+    const sessionMetadataOptions =
+      defaultMetadataOptions.sessionMetadataOptions;
+    let option: keyof SessionMetadataOptions;
+    for (option in defaultMetadataOptions.sessionMetadataOptions) {
+      sessionMetadataOptions[option] = checkboxValues.includes(option);
+    }
+
+    setMetadataOptions({
+      checkboxValues,
+      sessionMetadataOptions,
```

Comment:
No need to have `sessionMetadataOptions` in state. It can always be derived from `checkboxValues` during save.